scripts/reference-tables.template

```diff
@@ -52,6 +53,13 @@
         </tbody>
       </table>
     </section>
+    <section id="examples_experimental">
+      <h2 id="examples_experimental_label">Experimental Examples</h2>
+      <div><strong>NOTE:</strong> The HC abbreviation means example has High Contrast support.</div>
+      <ul id="examples_experimental_ul">
+
+      </ul>
+    </section>
```

Comment:
With no experimental examples, this would just be an empty section with a potentially unrelated note. Any thoughts given to this? Should the emptiness be filled with some other note? Should the section be conditionally shown?

Acknowledging it may also not matter at all.

Reply:
Closing the loop on this, it was discussed in the [May 7th meeting](https://www.w3.org/2024/05/07-aria-apg-minutes.html#t07) and the decision was to:

> [...] change the script to only include that section if there is at least one piece of experimental content [...]

Reply:
I believe this is handled. Thank you!